client/src/components/Overview/AddToCart.jsx

```diff
@@ -28,33 +31,29 @@ function AddToCart({ skus }) {
   }
 
   useEffect(() => setSelectedSku(null), [skus]);
-
-  useEffect(() => {
-    setQuantity(1);
-    setError(false);
-  }, [selectedSku]);
+  useEffect(() => setQuantity(1), [selectedSku]);
```

Comment:
I also just learnt that you can use an empty useEffect just to track update for dependency. I will steal this and upload my code tmr to make less API calls.